src/esp/gfx/LightSetup.cpp

```diff
@@ -15,32 +15,63 @@ bool operator!=(const LightInfo& a, const LightInfo& b) {
   return !(a == b);
 }
 
-Magnum::Vector3 getLightPositionRelativeToCamera(
+Magnum::Vector4 getLightPositionRelativeToCamera(
     const LightInfo& light,
     const Magnum::Matrix4& transformationMatrix,
     const Magnum::Matrix4& cameraMatrix) {
+  ASSERT(light.position.w() == 1 || light.position.w() == 0);
+  const auto pos = light.position.xyz();
+
+  bool isDirectional = light.position.w() == 0;
+  Magnum::Vector4 out{0};
+
   switch (light.model) {
     case LightPositionModel::OBJECT:
-      return transformationMatrix.transformPoint(light.position);
+      out.xyz() = isDirectional ? transformationMatrix.transformVector(pos)
+                                : transformationMatrix.transformPoint(pos);
+      break;
     case LightPositionModel::GLOBAL:
-      return cameraMatrix.transformPoint(light.position);
+      out.xyz() = isDirectional ? cameraMatrix.transformVector(pos)
+                                : cameraMatrix.transformPoint(pos);
+      break;
     case LightPositionModel::CAMERA:
-      return light.position;
+      out.xyz() = pos;
+      break;
   }
 
-  CORRADE_INTERNAL_ASSERT_UNREACHABLE();
+  out.w() = light.position.w();
+  return out;
 }
 
 LightSetup getLightsAtBoxCorners(const Magnum::Range3D& box,
-                                 const Magnum::Color4& lightColor) {
+                                 const Magnum::Color3& lightColor) {
   // NOLINTNEXTLINE(google-build-using-namespace)
   using namespace Magnum::Math::Literals;
 
-  return LightSetup{
-      {box.frontTopLeft(), lightColor},    {box.frontTopRight(), lightColor},
-      {box.frontBottomLeft(), lightColor}, {box.frontBottomRight(), lightColor},
-      {box.backTopLeft(), lightColor},     {box.backTopRight(), lightColor},
-      {box.backBottomLeft(), lightColor},  {box.backBottomRight(), lightColor}};
+  constexpr float w = 1;
```

Comment:
Again, there's no obvious conversion from our previous hybrid lights to our new point and directional lights. This change, along with the change in the header to use a default color intensity of 10, produces reasonable results in the zatun scene.

@aclegg3 , maybe you'd like to weight in. We can simply replace this function with something that provides reasonable default lighting. This would probably be 2-3 directional lights to approximate something like cloudy daylight lighting.